src/PostgREST/QueryBuilder.hs

```diff
@@ -16,13 +16,13 @@ module PostgREST.QueryBuilder (
   , readRequestToCountQuery
   , requestToCallProcQuery
   , limitedQuery
-  , setLocalQuery
-  , setLocalSearchPathQuery
+  , setConfigLocal
   ) where
 
 import qualified Data.ByteString.Char8           as BS
 import qualified Data.Set                        as S
 import qualified Hasql.DynamicStatements.Snippet as H
+import           PostgREST.Private.Common
```

Comment:
I always wonder where to put my `import`s. Looking at the existing code base, my understanding is, that we have 3 groups of imports:
- import qualified
- import (...)
- import

This belongs in the last category, so further down, right?

Reply:
Same on all the other imports, they look random to me :).

Reply:
@wolfgangwalther You're right, I used to pay more attention to the imports up until the introduction of `stylish-haskell`, I thought it took full care of them. I guess it doesn't order them alphabetically.